crates/bevy_render/src/mesh/shape/mod.rs

```diff
@@ -187,33 +187,70 @@ impl From<Quad> for Mesh {
 pub struct Plane {
     /// The total side length of the square.
     pub size: f32,
+    /// The number of subdivisions in the mesh.
+    /// 0 is the original plane geometry.
+    /// 1 is one line in both the X direction and the Z direction splitting the plane resulting in a plane with 4 quads / 8 triangles.
+    /// > 1 is multiple splits.
+    pub subdivisions: u32,
```

Comment:
These doc strings aren't totally clear about the number of segments produced at higher subdivisions.

My intuition would be that this number reflects the number of evenly sized segments the plane contains.

Reply:
`(n + 1)^2`